setup.py

```diff
@@ -41,6 +41,8 @@
         "eth-hash>=0.1.0,<1.0.0",
         "toolz>0.8.2,<1;implementation_name=='pypy'",
         "cytoolz>=0.8.2,<1.0.0;implementation_name=='cpython'",
+        "mypy<0.600",
+        "typed-ast<1.2.0,>=1.1.0",
```

Comment:
What's with this requirement?  Haven't seen it elsewhere.

Reply:
@pipermerriam thanks for the review! This is what's causing `pypy3` env to fail on circle - and I tried pinning it to a range of versions, but that didn't fix it. It's taken out now, but any thoughts as to how to fix Circle?